tests/core/device-reg/task.yaml

```diff
@@ -29,6 +26,9 @@ execute: |
     snap model --serial --assertion | MATCH "authority-id: canonical"
     snap model --serial --assertion | MATCH "brand-id: canonical"
     case "$SPREAD_SYSTEM" in
+        ubuntu-core-20-64)
```

Comment:
We should specify here the values for ubuntu-core-20-arm-*) as well.

Reply:
Do we know what the models will be named like? Will it be:

```
            snap model --serial --assertion | MATCH "model: ubuntu-core-20-$gadget_name"
```

like the UC18 case?

Reply:
@anonymouse64 sorry but I don't know, perhaps either @mvo5 or @pedronis know?

Reply:
I have disabled this test for uc20 arm deivces for now, we can re-enable when we know what the model names look like.